test/data/co2_PR_example.cti

```diff
@@ -0,0 +1,168 @@
+# Transport data from file ../transport/gri30_tran.dat.
```

Comment:
This file isn't used anywhere, and this format is not implemented for Peng-Robinson, right? Can it be deleted?

Reply:
Yes, we can delete, but the `cti` format is supported, which I think relates to your comment above about `ctml_writer.py` - adding support for the acentric factor allows a user to implement via cti file, if they so choose.

Reply:
I don't think you can create a `PengRobinson` instance from XML, there doesn't appear to me to be any code in `PengRobinson.cpp` that reads the XML nodes to create the phase. In any case, I believe @speth suggested that this class only be available from YAML, since XML/CTI are planned to be deprecated starting in 2.5.0 (although no warnings will be shown until a later version). Last, this file doesn't appear to be used anywhere; if there is support for the CTI format, it should be tested somewhere.

Reply:
Removed this file since it wasn't used anywhere.